examples/hitl/rearrange_v2/ui.py

```diff
@@ -486,13 +491,32 @@ def _update_hovered_object_ui(self):
                 sim, object_id, world._link_id_to_ao_map
             )
             if obj is not None:
+                object_category = world.get_category_from_handle(obj.handle)
+
+                obj_states = world.get_states_for_object_handle(obj.handle)
+                for state in obj_states:
+                    spec = state.state_spec
+                    if isinstance(spec, BooleanObjectState):
+                        val = cast(bool, state.value)
+                        object_states.append(
+                            (
+                                spec.display_name,
+                                "True" if val else "False",
+                            )
+                        )
+                    else:
+                        # Unsupported type.
+                        pass
+
                 primary_region = world.get_primary_object_region(obj)
                 if primary_region is not None:
                     primary_region_name = primary_region.category.name()
 
-        if primary_region_name is not None:
+        if primary_region_name is not None and object_category is not None:
             # TODO: Draw UI
-            # print(primary_region_name)
+            # print(f"Region: {primary_region_name}")
+            # print(f"Category: {object_category}")
+            # print(f"States: {object_states}")
```

Comment:
remove these prints altogether or put them under debug/verbose flag?

Reply:
This is going to be replaced by a GUI in a follow-up PR.